client/src/pages/Contest/Contest.tsx

```diff
@@ -0,0 +1,202 @@
+    },
+    '.MuiImageListItemBar-titleWrap': {
+      color: '#333',
+      marginTop: '12px',
+    },
+    '.MuiImageListItemBar-title': {
+      fontSize: '15px',
+      fontWeight: 'bolder',
+      textDecoration: 'underline',
+    },
+    '.MuiImageListItem-imgFullHeight': {
+      opacity: '0.8',
+    },
+  },
+})(() => null);
+
+export default function Contest(): JSX.Element {
+  const classes = useStyles();
+  const [value, setValue] = useState(0);
+  //const [contestSubmissions, setContestSubmissions] = useState<Submission[]>([]);
+  const [contestSubmissions, setContestSubmissions] = useState<any>([]);
+  const { loggedInUser } = useAuth();
+
+  const newTheme = createMuiTheme({
+    palette: {
+      primary: {
+        main: '#000000',
+      },
+      secondary: {
+        main: '#3d3d3d',
+      },
+    },
+  });
+
+  interface TabPanelProps {
+    direction?: string;
+    value: number;
+    children: React.ReactNode;
+    index: number;
+  }
+
+  useEffect(() => {
+    async function getContestAllSubmissions() {
+      try {
+        const contestAllSubmissions = await getContestById();
+        if (contestAllSubmissions) {
+          setContestSubmissions(contestAllSubmissions);
+        }
+      } catch (err) {
+        console.log(err);
+      }
+    }
+    getContestAllSubmissions();
+  }, []);
+
+  const handleChange = (event: React.ChangeEvent<Record<string, unknown>>, valueChange: number) => {
+    setValue(valueChange);
+  };
+
+  const Panel = function (props: TabPanelProps) {
+    return (
+      <div role="tabpanel" id={`${props.index}`}>
+        {value === props.index && (
+          <Container>
+            <Typography>{props.children}</Typography>
+          </Container>
+        )}
+      </div>
+    );
+  };
+
+  const capitalize = (s: string) => s && s[0].toUpperCase() + s.slice(1);
+
+  return loggedInUser ? (
+    <>
+      <CssBaseline />
+      <GlobalCss />
+      <AuthHeader linkTo="/signup" btnText="sign up" />
+      <Container className={classes.container}>
+        <Grid className={classes.grid} container style={{ marginBottom: '35px' }}>
+          {/* has to back to contests list, no route for that yet, so using dashboard for testing*/}
+          <Grid item>
+            <Typography>
+              <Link to="/dashboard" className={classes.greyText}>
+                <ArrowBackIosIcon className={classes.backIcon} />
+                Back to contests list
+              </Link>
+            </Typography>
+          </Grid>
+        </Grid>
+        <Grid className={classes.grid} container>
+          <Grid item xs={10}>
+            <Typography className={classes.contestTitle}>
+              {contestSubmissions.title ? contestSubmissions.title : 'Lion tattoo concept in minimal style'}{' '}
+              <Button className={classes.prizeAmount}>
+                <Typography className={classes.prize}>
+                  {contestSubmissions.prizeAmount ? contestSubmissions.prizeAmount : '$150'}
```

Comment:
What is the reason for using ternary operators to render alternate data?

Reply:
did not want to render empty therefore, used ternary operators to show if the view generated is matching the mock. Will be fixed once, fetching from backend is successful. I will fix it :)